drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -516,11 +516,14 @@ def _get_item_stats(self):
         item_state = self._get_item_state()
 
         required_items = [str(item['id']) for item in all_items if self._get_item_zones(item['id']) != []]
-        placed_items = [item for item in required_items if item in item_state]
-        correct_items = [item for item in placed_items if item_state[item]['correct']]
+        required_items_placed = [item for item in required_items if item in item_state]
+        required_items_correct = [item for item in required_items_placed if item_state[item]['correct']]
 
-        required_count = len(required_items)
-        correct_count = len(correct_items)
+        decoy_items = [str(item['id']) for item in all_items if self._get_item_zones(item['id']) == []]
+        decoy_items_placed = [item for item in decoy_items if item in item_state]
+
+        required_count = len(required_items) + len(decoy_items)
+        correct_count = len(required_items_correct) + len(decoy_items) - len(decoy_items_placed)
```

Comment:
@arbrandes nit: the way it written right now is a bit confusing - I even started writing "it wouldn't work like expected ...", got to the part where I'm providing an example and than realized that it actually will.

To avoid that kind of confusion, could you please update it to read more like in SOL-1988:  [(# of non-decoy items placed onto a correct zone) + (# of decoy items correctly left unplaced)] ÷ (total # of items), i.e

    decoy_items_in_bank = [item for item in decoy_items if item **not** in item_state]

    correct_count = len(required_items_correct) + len(decoy_items_in_bank)